pegasus/sites.v3/code.org/public/curriculum/csf.haml

```diff
@@ -115,4 +117,11 @@ theme: responsive_full_width
   .wrapper
     = view :subscribe_form, subscribe_form_section_title: hoc_s(:subscribe_form_section_title), subscribe_form_section_desc: hoc_s(:subscribe_form_section_desc_csf), form_height: "375px", form_src: "http://go.pardot.com/l/153401/2018-10-02/lzp5jd"
 
+%section.bg-neutral-light
+  .wrapper
+    -# The section is using the new namespace pattern for strings
+    %h2=hoc_s("csf_page.additional_resources.heading")
+    %p=hoc_s("csf_page.additional_resources.desc")
```

Comment:
Should these be centered like in the Figma?

Reply:
Ah good question! I'm going to leave it left-aligned since there are no other centered sections on this page.